### ISSUE TITLE
Realtime updates (using SSE, WS or polling etc)

### ISSUE THREAD

**Issue by minimal** [AndreaCrotti/elo#77]
It seems like you have to refresh to see if anyone else has added results since you loaded the page. Live updates with server side events or similar would be nice

**Comment by AndreaCrotti** [comment]
Good idea, I've never done so more concrete suggestions are welcome of course.
